test/new_tests/test_query.py

```diff
@@ -1140,3 +1140,7 @@ def callback(input_tuple):
             
         assert records
         assert len(records) == 3
+
+    def test_query_with_base64_cdt_ctx(self):
```

Comment:
get_cdt_ctx_base64 should have its own test file, similar to test_get_expression_base64.py. This functionality needs more testing. Please include some negative tests aswell.